### PR TITLE
Add functions getCanvas and getImage

### PR DESCRIPTION
**Pull request by cainmartin**
Added new methods getCanvas and getImage to handle the situation where Jeff is used inside an electron application.

### CODE REVIEW

File: src/CanvasRenderer/Image.js [hunk 1]
```diff
@@ -1,8 +1,21 @@
-var Image;
-if (typeof(window) !== 'undefined' && window.Image) {
-	Image = window.Image;
```
Comment (bchevalier) [inline src/CanvasRenderer/Image.js]: Actually this worked

File: src/CanvasRenderer/Canvas.js [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+	} else {
+		Canvas = require('canvas');
+
+		return new Canvas(w, h);
+	}
 }
-module.exports = Canvas;
\ No newline at end of file
+
+module.exports = getCanvas;
```
Comment (bchevalier) [inline src/CanvasRenderer/Canvas.js]: may be the file can get rename "getCanvas" too

**Comment by cainmartin** [comment]
Please check latest changes as per your comments.

**Comment by bchevalier** [comment]
lgtm

**Pull request merged**